addon/adapters/application.js

```diff
@@ -42,21 +42,30 @@ export default Ember.Object.extend(FetchMixin, Evented, {
     Find resource(s) using an id or a using a query `{id: '', query: {}}`
 
     @method find
-    @param {Object|String} options use a string for a single id or an object
+    @param {Object|String|Number} options use a string for a single id or an object.
     @return {Promise}
   */
   find(options) {
-    if (typeof options === 'string') {
-      return this.findOne(options);
-    } else if (typeof options === 'object') {
-      if (options.id) {
-        return this.findOne(options.id, options.query);
+    // Collect id and query from options (if given).
+    // ids are cast to string to allow number input.
```

Comment:
perhaps instead of "to allow" we could say to "ensure id is a Strings (which spec requires)"